lib/MqttClient.js

```diff
@@ -437,9 +437,19 @@ MqttClient.prototype.handleCustomCommand = function (message) {
                     const areas = this.configuration.get("areas");
                     let zones = [];
 
-                    msg.zone_ids.forEach(function (zone_id) {
+                    msg.zone_ids.forEach(function (zone_selector) {
+                        let zone_id = typeof zone_selector === 'object' ? zone_selector.id : zone_selector;
+                        let repeats = Number(zone_selector.repeats);
                         let area = areas.find(e => Array.isArray(e) && e[0] === zone_id);
                         if (area && Array.isArray(area[1])) {
+                            if (!isNaN(repeats)) {
+                                if (repeats >= 1 && repeats <= 3) {
+                                    console.info(new Date(), "Zone cleanup repeats override for zone '" + zone_id + "': " + repeats);
+                                    area[1][4] = repeats;
```

Comment:
Btw I'm afraid this way you're overwriting the number of iterations in `this.configuration.get("areas")`.
This is how javascript works with complex objects.

Reply:
Fixed, hopefully. Really not used to JS...